src/rust/Cargo.toml

```diff
@@ -18,7 +18,7 @@ rustc-simple-version = "=0.1.0"
 # will complain if it does not match)
 rand = "=0.8.5"
 
-itertools = "=0.11.0"
+itertools = "=0.10.5"
```

Comment:
This doesn't have any risk because itertools is only used when tracing is enabled.